activities/titilayo_tl.json

```
@@ -0,0 +1,20 @@
+{
+  "tasks": [
+    {
+      "layers": [
+        ["d.rast", "map=rural_streams"], ["raster_series", "series_map=flooding"]
```

`raster_series` won't work here so I suggest just displaying the final flood map. On Tangible Landscape, we can open the GRASS timeseries animation window and see the flood animation.

```suggestion
        ["d.rast", "map=flooding_5.0"], ["d.rast", "map=streams"]
```